CRM/Event/Form/Task/Batch.php

```diff
@@ -260,14 +260,10 @@ public function postProcess() {
    * @param int $participantId
    * @param int $statusId
    *
-   * @return mixed
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
   public static function updatePendingOnlineContribution($participantId, $statusId) {
-    if (!$participantId || !$statusId) {
```

Comment:
moved to the calling function - don't call a function if you don't have the variables